src/Features/CSharp/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -59,6 +59,11 @@ internal partial class CSharpIntroduceVariableService
                 return Task.FromResult(IntroduceLocalDeclarationIntoLambda(
                     document, expression, newLocalName, declarationStatement, parentLambda, allOccurrences, cancellationToken));
             }
+            else if (IsInExpressionBodiedMember(expression))
```

Comment:
What about blocks in expression-bodied members that aren't lambdas? What happens with:

```C#
    Func<int, int> X() => delegate (int y)
    {
        return 8;
    };
```